### DISCUSSION TITLE
Deactivating and editing the blog type selector

### DISCUSSION THREAD

**Discussion by Daeraxa** [vuepress-theme-hope/vuepress-theme-hope#2402] [Q&A] [answered]
Is there a plugin config option to configure the article-type-wrapper in the theme's default blog implementation?
![image](https://user-images.githubusercontent.com/58074586/201356784-a1d79774-0683-47fc-b890-e73210893b92.png)

**Comment by Mister-Hope** [comment]
Nope, but we could add one if possible.

Is there any strong reasons that you dislike it?

I understand that you may not need slide page and encryption page, so we could make some improvements in the next coming versions if none of the articles are about them.

**Reply by Daeraxa** [reply]
So for our use we only need relatively simple features as a "dev blog" for an open source project so I don't see us needing the slide or encryption ones (at least not yet). If I could turn off slides and encryption then I would absolutely keep the selector.
I do like the selector itself but it would be great if we could set each type or path individually.
So instead of `Article`, `Star`, `Slide`, `Encryption`. I could have something like:
![image](https://user-images.githubusercontent.com/58074586/201373328-536af89d-0971-4b57-b8fb-3a99038b4466.png)
Where each one could be set by category (e.g. `Important` might just be an alias for `Star`, `Releases` could be `tag/release`, and dev log might be a type that is `category/dev` & `Author A || Author B || Author C`
The above are just examples using the different features I know the blog plugin already supports.
Because the website itself is mainly for documentation and a homepage the blog isn't *as* important so having that selector instead of navbar or sidebar items for different filters or categories would be really nice.

**Comment by Mister-Hope** [comment, marked as answer]
Please track https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/2450.

Sorry this is a bit more complicated so we do not have roadmap for this.
